cmd/tarmak/cmd/cluster_logs.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Jetstack Ltd. See LICENSE for details.
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/jetstack/tarmak/pkg/tarmak"
+)
+
+// clusterLogsCmd handles `tarmak clusters logs`
+var clusterLogsCmd = &cobra.Command{
+	Use:   "logs",
+	Short: "Gather logs from an instance pool",
+	Run: func(cmd *cobra.Command, args []string) {
+		t := tarmak.New(globalFlags)
+		defer t.Cleanup()
+
+		if len(args) == 0 {
+			t.Log().Fatal("expecting at least a one instance pool name")
+		}
+
+		t.CancellationContext().WaitOrCancel(t.NewCmdTerraform(args).Logs)
```

Comment:
Why `NewCmdTerraform`? I don't feel this belongs here as the logs have nothing to do with Terraform.

Reply:
I agree however neither is SSH(). I like using this struct for consistency and using the context etc. Perhaps we should rename it to CmdTarmak however this doesn't make much sense being in terraform.go
Thoughts?